demos/wasi/hello-world-rust/README.md

```diff
@@ -0,0 +1,47 @@
+# Hello World Rust for WASI
+A simple hello world example in Rust that will print the environment variables
```

Comment:
nit: This sentence is a little confusing. Maybe break into a bullet list?

Reply:
Good point. Breaking out